arxivscraper/arxivscraper.py

```diff
@@ -88,6 +88,8 @@ class Scraper(object):
     they were created outside of the given date range. Default: today.
     t: int
     Waiting time between subsequent calls to API, triggred by Error 503.
+	timeout: int 
+    Timeout in seconds after which the scraping stops.
```

Comment:
Adding Default: XX seconds.

Reply:
Done. Increased the timeout to 5 minutes and added to docstring.